### PR TITLE
Humble

### PR DESCRIPTION
**Pull request by tianjiawen1**

### CODE REVIEW

File: net_ft_description/urdf/net_ft_sensor.urdf.xacro [hunk 1]
```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0"?>
 <robot xmlns:xacro="http://wiki.ros.org/xacro" name="net_ft_sensor">
     <!-- parameters -->
-    <xacro:arg name="ip_address" default="192.168.1.1" />
+    <xacro:arg name="ip_address" default="192.168.1.10" />
```
Comment (gbartyzel) [inline net_ft_description/urdf/net_ft_sensor.urdf.xacro:4]: The default IP is `192.168.1.1`, so please, revert this :)

File: net_ft_driver/launch/net_ft_broadcaster.launch.py [hunk 2]
```diff
@@ -135,7 +135,7 @@ def generate_launch_description():
     declared_arguments.append(
         launch.actions.DeclareLaunchArgument(
             name="ip_address",
-            default_value="192.168.1.1",
+            default_value="192.168.1.10",
```
Comment (gbartyzel) [inline net_ft_driver/launch/net_ft_broadcaster.launch.py:138]: Also, here

File: net_ft_driver/launch/net_ft_broadcaster.launch.py [hunk 3]
```diff
@@ -159,7 +159,7 @@ def generate_launch_description():
     declared_arguments.append(
         launch.actions.DeclareLaunchArgument(
             name="use_hardware_biasing",
-            default_value="0",
+            default_value="flase",
```
Comment (gbartyzel) [inline net_ft_driver/launch/net_ft_broadcaster.launch.py:162]: There is a typo: should be `false`

**Review by gbartyzel — CHANGES_REQUESTED** [review]

**Comment by gbartyzel** [comment]
@tianjiawen1 Can you look at the comments?

**Comment by tianjiawen1** [comment]
> @tianjiawen1 Can you look at the comments?

yes，i tested on ati gamma ,it works.

**Comment by gbartyzel** [comment]
@tianjiawen1 That's great, but please change the default IP address to 192.168.1.1 (it's the default for ATI AXIA and more)

**Comment by gbartyzel** [comment]
Fix introduced in #14. Closing PR/

**Pull request closed without merging**